pgctl/fuser.py

```diff
@@ -22,7 +26,7 @@ def stat(path):
         trace('fuser suppressed: %s', error)
         return None
     else:
-        return (path.st_ino, path.st_dev)
+        return StatTuple(path.st_ino, path.st_dev, path.st_nlink)
```

Comment:
Why use this over the existing tuple? E.g. is it just to reduce the information we pass around?